src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -312,7 +312,19 @@ public static void writeBundle(OutputStream outputStream, final List<Component>
                     }
                     final String name = getNameFiltered(maybeFilter, content.getName(), content.getFilterableParameters());
                     final ZipArchiveEntry entry = new ZipArchiveEntry(name);
-                    entry.setTime(content.getTime());
+                    
+                    try {
```

Comment:
It happens only with contents based on Files and it's due to a lack of permissions if I understood correctly. For me, it's better to just move the instruction into the existing `try {` that avoids breaking the loop if something wrong happens with a specific content.

Reply:
@MRamonLeon if an exception is raised before doing `binaryOut.putArchiveEntry(entry)`, the method `binaryOut.closeArchiveEntry()` will fail:

```
                    try {
                        // if an exception is raised here
                        binaryOut.putArchiveEntry(entry); 
                        [...]
                    } catch (Throwable e) {                        
                        [...]
                    } finally {
                        [...]
                        binaryOut.closeArchiveEntry(); // This breaks with `java.io.IOException: No current entry to close`
                    }
```

Reply:
If the exception is raised inside the method (which is possible) it's going to fail as well. I suppose `bynaryOut.closeArchiveEntry();` should go at the end of the `try` section, but then the two previous sentences should go there too. The logic about these streams is very messy and deserves a bit of clean up. My concern is that if we cannot get the time of the entry, we are not going to be able to get the full content of that. Therefore, my proposal to include it all together in the same `try`. I've seen some implementation encosing the closeArchiveEntry in a try-catch clause, but it's almost the same. So I keep it up to you.

Reply:
> My concern is that if we cannot get the time of the entry, we are not going to be able to get the full content of that.

You are right. I think in most cases, if we cannot get the time we cannot get the content. 

****

Looking into this, the `putArchiveEntry` is gonna do a `closeArchiveEntry` if needed. So we actually don't need to do the `binaryOut.closeArchiveEntry()` in that finally as it would be taken care of by the next iteration if required.